### PR TITLE
Add ARO test in 4.10

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-azure-aro-p1-f1

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-azure-aro-p1-f1

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-4.10-upgrade-from-stable-4.10-azure-aro-cert-rotation-p1-f1

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml [hunk 1]
```diff
@@ -766,6 +766,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private-workers-marketplace
+- as: azure-aro-p1-f1
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:769]: daily job here?

**Comment by jianlinliu** [comment]
lgtm.

cc @mgahagan73 here to review.

**Comment by jianlinliu** [comment]
/lgtm

so that get it merged asap to catch up release date.

**Pull request merged**